src/cephadm/cephadm

```diff
@@ -3591,13 +3591,6 @@ def prepare_ssh(
 
     cli(['config-key', 'set', 'mgr/cephadm/ssh_user', ctx.ssh_user])
```

Comment:
I think this needs to be changed to do 'ceph cephadm set-user ...', like the 'set-ssh-config' etc commands further down.

Reply:
Can you give me a little more context of what we're trying to do with this change? If the current usage is an issue, I'll be happy to take it up and make the required changes.

Reply:
I think we want
```
cli(['cephadm', 'set-ssh-user', ctx.ssh_user'])
```
since it is happening after the cephadm module is enabled.